README.rst

```diff
@@ -205,6 +205,8 @@ These parameters are common to all master key providers:
   key provider. ``The discovery`` attribute is only available if you are using an ``aws-kms`` provider.
 
     * If using ``aws-kms`` to decrypt, `you must specify either a key or discovery with a value of true`_.
+    * If using ``aws-kms`` to decrypt and specifying a key, decryption will only be attempted for keys identified with
+      a key ARN.
```

Comment:
```suggestion
    * If using ``aws-kms`` to decrypt and specifying a key, you must use a key ARN; key ids, alias names, and alias ARNs are not supported.
```
I found this a bit hard to parse. I would use the same wording you use in the other code bases.